scripts/zkai_d64_rmsnorm_swiglu_statement_fixture.py

```
@@ -0,0 +1,518 @@
+        if json_path is not None:
+            json_path.parent.mkdir(parents=True, exist_ok=True)
+            json_path.write_text(json.dumps(payload, indent=2, sort_keys=True) + "\n", encoding="utf-8")
+        if tsv_path is not None:
+            tsv_path.parent.mkdir(parents=True, exist_ok=True)
+            with tsv_path.open("w", encoding="utf-8", newline="") as handle:
+                writer = csv.DictWriter(handle, fieldnames=TSV_COLUMNS, delimiter="\t", lineterminator="\n")
+                writer.writeheader()
+                for row in rows_for_tsv(payload):
+                    writer.writerow(row)
+    except OSError as err:
+        raise StatementFixtureError(f"failed to write statement fixture output: {err}") from err
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)